launch

```diff
@@ -570,9 +570,6 @@ def main():
   if not ret:
     return 1
 
-  if not build_brackets():
```

Comment:
I don't understand why you move the grunt build to the rear.
Is not setting for grunt path at the 569 line?

Reply:
grunt path is setted in 596 line.

Reply:
Oh, I see.